src/xcode/ENA/ENA/Source/HTTPClient/Resources/TeleTan/Locator+TeleTan.swift

```diff
@@ -10,7 +10,7 @@ extension Locator {
 	// receive:	JSON
 	// type:	default
 	// comment:
-	static func registrationToken(
+	static func teleTan(
```

Comment:
How was the decision about that naming? The path says "registrationToken"...

Reply:
Good catch! There was a renaming failure. registrationToken and teleTan got swapped. I corrected this.